src/simulation/environment/spiceInterface/spiceInterface.cpp

```diff
@@ -47,18 +56,14 @@ SpiceInterface::SpiceInterface()
 
     referenceBase = "j2000";
     zeroBase = "SSB";
-	timeOutPicture = "MON DD,YYYY  HR:MN:SC.#### (UTC) ::UTC";
+    timeOutPicture = "MON DD,YYYY  HR:MN:SC.#### (UTC) ::UTC";
 
     //! - set default epoch time information
     char string[255];
     snprintf(string, 255, "%4d/%02d/%02d, %02d:%02d:%04.1f (UTC)", EPOCH_YEAR, EPOCH_MONTH, EPOCH_DAY, EPOCH_HOUR, EPOCH_MIN, EPOCH_SEC);
     this->UTCCalInit = string;
-
-    return;
 }
 
-/*! The only needed activity in the destructor is to delete the spice I/O buffer
```

Comment:
Why are you removing all the method descriptions?  Please add them back to ensure the BSK documentation shows descriptions of these methods.